samples/ConsoleSample/main.py

```diff
@@ -259,12 +286,17 @@ def sendProtoOACancelOrderReq(orderId, clientMsgId = None):
         "NewLimitOrder": sendNewLimitOrder,
         "NewStopOrder": sendNewStopOrder,
         "ClosePosition": sendProtoOAClosePositionReq,
-        "CancelOrder": sendProtoOACancelOrderReq}
+        "CancelOrder": sendProtoOACancelOrderReq,
+        "DealOffsetList": sendProtoOADealOffsetListReq,
+        "GetPositionUnrealizedPnL": sendProtoOAGetPositionUnrealizedPnLReq,
+        "OrderDetails": sendProtoOAOrderDetailsReq,
+        "OrderListByPositionId": sendProtoOAOrderListByPositionIdReq,
+    }
 
     def executeUserCommand():
         try:
             print("\n")
-            userInput = inputimeout("Command (ex help): ", timeout=18)
+            userInput = inputimeout("Command (ex help): ", timeout=180)
```

Comment:
Setting timeout too high will cause disconnection because it will keep the thread busy and while waiting for user input we can't send heartbeat to server, please set it back to 18 or find a better way to handle this case.

Reply:
done